doctr/datasets/vocabs.py

```diff
@@ -24,3 +29,4 @@
 VOCABS['portuguese'] = VOCABS['english'] + 'áàâãéêëíïóôõúüçÁÀÂÃÉËÍÏÓÔÕÚÜÇ' + '¡¿'
 VOCABS['spanish'] = VOCABS['english'] + 'áéíóúüñÁÉÍÓÚÜÑ' + '¡¿'
 VOCABS['german'] = VOCABS['english'] + 'äöüßÄÖÜẞ'
+VOCABS['arabic'] = VOCABS['arabic_letters'] + VOCABS['arabic_farsi_letters'] + VOCABS['hindi_numbers'] + VOCABS['digits'] + VOCABS['arabic_diacritics'] + VOCABS['arabic_unique_punctuation'] + VOCABS['punctuation']
```

Comment:
One last thing: we have to make sure that there is no character duplicate (cf. my previous comment). Could you make sure that this is the case?

Reply:
To make flake8 content:
```python
VOCABS['arabic'] = (VOCABS['digits'] + VOCABS['hindi_digits'] + VOCABS['arabic_letters'] + VOCABS['persian_letters'] +
    VOCABS['arabic_diacritics'] + VOCABS['arabic_punctuation'] + VOCABS['punctuation'])
```